src/redux/categories/categories.js

```diff
@@ -0,0 +1,21 @@
+// Actions
+const STATUS = 'STATUS';
```

Comment:
- [ ] Please would you kindly check the comment above? :arrow_up: Making sure that the values of the action types follow the duck pattern ensures that we can easily scale our app in the future and also facilitate debugging our app when it becomes larger/bulky.

Reply:
Thanks! It's done